packages/material-ui/src/Popper/Popper.d.ts

```diff
@@ -17,8 +17,11 @@ export type PopperPlacementType =
   | 'top-start'
   | 'top';
 
+export interface PopperRef {
```

Comment:
I'm sure we can do better here, using https://github.com/FezVrasta/popper.js/blob/8f2469111dcc483623a5fcba99e34c661f9ea74d/packages/popper/index.d.ts#L155. But how? 🤔

Reply:
If you don't need something specific you can just say it extends Popper
If you need specific things from it, use `export type PopperRef = Pick<Popper, 'update' | 'some other prop'>`

Reply:
I believe we can expose everything. We already do it with: `popperOptions`.

Reply:
Does it work with d719f75ece12a79d85a8aebbdcec1131c9b02689?